### ISSUE TITLE
Theme design

### ISSUE THREAD

**Issue by azubieta** [ds9-artwork/sddm-theme-pss#1]
@UriHerrera we need a mockup for the sddm theme. Somthing like the existent [maui theme](https://www.youtube.com/watch?v=dJ28mrOeuNA)

**Comment by star-buck** [comment]
We dont need a mockup for this, the idea is to take the code and make it
look okay as is.

It has fx and random script to create background, just look at all elements
and make them look nice by centering, aligning, etc. placing them common
sense.

**Comment by azubieta** [comment]
Ok, I already made it work Just didn't like how it looks but I'll improve it.

I'll take the current maui theme as guide for positioning the different elements.

**Comment by UriHerrera** [comment]
The current Maui SDDM theme uses a library called [fluid](https://github.com/hawaii-desktop/fluid) and its SDDM theme looks like this (at least the recent version that it's part of Hawaii and that it's installed from the AUR packages):

![Maui SDDM Theme](http://i.imgur.com/Lptd0rF.png)

> Maui SDDM Theme

I think this is doable without the need to use that specific library.

**Comment by star-buck** [comment]
Just position everything centered.
You could make a screen of how it currently looks and post it here, then we
decide what to do with it.
Just for now lets not implement hawaii and stick to the revived old theme.

**Comment by azubieta** [comment]
This is what I have done. It is composed by two screens, the first one is a lock page whit some general info and the second is the login page with the login specific stuff. There is a transition between pages were the lock page slides up.

It requires to be polished a little bit, but I will wait for your opinions to continue.

![sddm_lock_screen](https://cloud.githubusercontent.com/assets/1138094/17591643/a21a04f8-5fa4-11e6-9582-1765aa0248ea.png)
![sddm_login_screen](https://cloud.githubusercontent.com/assets/1138094/17591644/a21dea14-5fa4-11e6-987f-31d98d04ceaf.png)

**Comment by star-buck** [comment]
Alexis:
The first part is okay, the second screen should exactly have the look and
feel of the old mauiproject, that is copy password field, dark grey
background, etc.
Just place the elements in the middle/center.

**Comment by azubieta** [comment]
Is something like this? It still requires to a circular list view for the users. 
![sddm_login_srceen_gray](https://cloud.githubusercontent.com/assets/1138094/17596730/56f7719a-5fb8-11e6-9e70-b8f28e83a1c8.png)

**Comment by star-buck** [comment]
yeah, let me see if i can screengrab how it was looking like when it was working...

**Comment by star-buck** [comment]
pretty close:
![greeter snapshot](https://cloud.githubusercontent.com/assets/840070/17600930/fe9c1192-6005-11e6-9df3-0e458994903f.png)

**Comment by star-buck** [comment]
of course tastes vary, but imo this minimalism and background is something that looks really nice.

**Comment by UriHerrera** [comment]
Right, but the password field is too wide; half that size would be better. The name is quite close to the password field too it should have a bit more padding at the bottom. And of course the session buttons are almost invisible.

For a touch of flare it'd be nice to have the clock at the top in the format of the Alexis' [first image](https://github.com/plasma-extensions/sddm-theme-core/issues/1#issuecomment-239175380).

**Comment by azubieta** [comment]
Here are two screenshots more, now the items are more spaced and centred. Also it only shows 3 users in the screen in order to not overload it too much.

Shoud I remove the session and layout pickers  and add the "<" ">" at the borders ? 

![sddm_login_srceen_gray_2](https://cloud.githubusercontent.com/assets/1138094/17634248/53f1fc56-6097-11e6-87d0-51787afa10df.png)
![sddm_login_srceen_gray_1](https://cloud.githubusercontent.com/assets/1138094/17634247/53ef2f44-6097-11e6-9558-e97ee8c3938b.png)

**Comment by star-buck** [comment]
looks good!
maybe you can round the edges of the avatars just a bit, i mean automatically for any avatar.

**Comment by azubieta** [comment]
Ok, I uploaded the code. 
The user picture is placed inside a frame image that depending on the system style could have the border rounded or straight (wen you try it on kubuntu by example it looks rounded). This is the default behavior of the sddm breeze theme, now in order to force the rounded edges I can replace this frame by a rectangle an clip its content but this would make a little less customizable, should I do it ?

**Comment by star-buck** [comment]
can you do a picture with the rounded edges?
lets alsod make the password field double as big again.

**Comment by azubieta** [comment]
Ok, i'll remove the frame and let only the user picture with the edges rounded.
Also the password field width will be doubled.
Please check the login error message.
I was considering to add a CapsLock enabled alert inside the password field as an alert icon with a tooltip message, what do you think?

**Comment by azubieta** [comment]
I have been working on the rounded edges. Right now I can only fake it with a rounded rectangle or an image. Results that QtQuick Item class does support clipping but this is limited to rectangular shapes.

**Comment by star-buck** [comment]
Thats okay for now then.
I suggest to lrave it there and continue with starting the
notification-sidebar next.

**Comment by azubieta** [comment]
Are the notification-sidebar and the quick-panel the same?
Would you please open an issue for it whit an small description in the corresponding repository?
